tracer/build/_build/Build.Steps.cs

```diff
@@ -586,6 +588,27 @@ void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath a
             CompressZip(SymbolsDirectory, WindowsSymbolsZip, fileMode: FileMode.Create);
         });
 
+    Target ExtractDebugInfoAndStripSymbols => _ => _
+        .After(ZipMonitoringHome)
```

Comment:
Should this run before the `ZipMonitoringHome` step?

Reply:
`ZipMonitoringHome` renames files (native loader and natrive tracer. I think it's better if we get the debug files with the same names after the renaming, WDYT ?

Reply:
done